extensions/ui/org.eclipse.smarthome.ui.paper/web-src/js/controllers.configuration.js

```diff
@@ -505,7 +505,7 @@ angular.module('PaperUI.controllers.configuration',
 		} else {
 		    thing.item = {};
         }
-		thingSetupService.update(thing, function() {
+		thingService.updateConfig({thingUID:thing.UID},thing.configuration, function() {
```

Comment:
There is no save action for the Eclipse JS addon, but if you use "Source, Format" manually I would assume that it adds spaces after a comma.